src/OpenSpaceToolkit/Astrodynamics/EventCondition.cpp

```diff
@@ -36,12 +37,23 @@ EventCondition::Criteria EventCondition::getCriteria() const
     return criteria_;
 }
 
+Real EventCondition::getTarget() const
+{
+    return target_;
+}
+
+Real EventCondition::evaluate(const VectorXd& aStateVector, const Real& aTime) const
+{
+    return compute(aStateVector, aTime) - target_;
```

Comment:
Does the order/sign of the substraction matter for any use case? 🤔

Reply:
Alright, LGTM. Checked different target values (positive, negative) at `isSatisfied` level